### PR TITLE
Add missing license

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

**Comment by dhoppe** [comment]
The build error should be fixed after merging #15.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (juniorsysadmin) [inline LICENSE]: You might need to handle this LICENSE file a bit differently

Comment (dhoppe) [inline LICENSE]: I just picked this from puppet-grafana and I am not sure if we set this value properly for our other modules.

Comment (dhoppe) [inline LICENSE]: @juniorsysadmin How should I handle this, since there is no real name of the author?

Comment (alexjfisher) [inline LICENSE]: You don't.  The Appendix is part of the full text of the apache license.  You're not supposed to insert the copyright holder into the appendix.  If you wanted to, you'd use the boilerplate notice (with copyright year and holder modified) in every source file, but virtually no one does this.

Comment (alexjfisher) [inline LICENSE]: See https://github.com/voxpupuli/puppet-archive/pull/237#discussion_r84910919

**Pull request merged**